awx/ui/src/util/simulationWorker.js

```diff
@@ -1,21 +1,22 @@
-/* eslint-disable no-undef */
-importScripts('https://d3js.org/d3-collection.v1.min.js');
```

Comment:
In a follow up PR (so we don't block this fix) would we want CI to error if it finds `importScripts`?

Reply:
No because you can use importScripts to pull in local JS files, which is part of the fix in this PR.